app/components/blocks/SiteFooter/LanguageSelector/LanguageSelector.test.ts

```diff
@@ -1,5 +1,5 @@
 import { mountSuspended } from '@nuxt/test-utils/runtime'
-import { supportedLocales } from '~/i18n/i18nUtils'
+import { supportedLocales } from 'root/i18n/i18nUtils'
```

Comment:
"With this new structure, the ~ alias now points to the app/ directory by default (your srcDir). This means ~/components resolves to app/components/, ~/pages to app/pages/, etc." - [See here](https://nuxt.com/docs/4.x/getting-started/upgrade#new-directory-structure)